update-versions.py

```diff
@@ -0,0 +1,184 @@
+# Copyright 2018 Google LLC
```

Comment:
Sounds good - will do. I'll leave the `Dockerfile` and `setup.py` changes though since they are related to `.circleci/config.yml` logic to extract the version.

Reply:
Yes, I agree that it makes sense to keep the change which extracts the versions up.